modules/gdscript/gdscript_parser.cpp

```diff
@@ -4058,7 +4055,8 @@ void GDScriptParser::_parse_class(ClassNode *p_class) {
 
 							member._export.type=Variant::DICTIONARY;
 
-						} else*/ {
+						} else*/
```

Comment:
Maybe kill that old commented out code above?

Reply:
I'd prefer leaving it as-is, it might be potentially useful later on.